script.js

```diff
@@ -1,25 +1,133 @@
+  cardTechs: ['html', 'css', 'javascript'],
+  source: 'https://github.com/RashidMenhas/My-Portfolio',
+  live: '#',
+},
+{
+  cardId: 2,
+  title: 'Tonic',
+  desc: 'A daily selection of privately personalized reads; accounts or sign-ups required.',
+  imgScr: 'images/cardimg2.png',
+  cardTechs: ['html', 'css', 'javascript'],
+  source: 'https://github.com/RashidMenhas/My-Portfolio',
+  live: '#',
+},
+{
+  cardId: 3,
+  title: 'Tonic',
+  desc: 'A daily selection of privately personalized reads; accounts or sign-ups required.',
+  imgScr: 'images/cardimg3.png',
+  cardTechs: ['html', 'css', 'javascript'],
+  source: 'https://github.com/RashidMenhas/My-Portfolio',
+  live: '#',
+},
+{
+  cardId: 4,
+  title: 'Tonic',
+  desc: 'A daily selection of privately personalized reads; accounts or sign-ups required.',
+  imgScr: 'images/cardimg5.png',
+  cardTechs: ['html', 'css', 'javascript'],
+  source: 'https://github.com/RashidMenhas/My-Portfolio',
+  live: '#',
+},
+];
 
-about.addEventListener('click', () => {
-  menuList.style.display = 'none';
+// Dynamic Card Section
+let cardSection = '';
+cards.forEach((card) => {
+  cardSection += `<div class="card">
+          <div class="card-heading">
+              <img src="${card.imgScr}" alt="card">
+          </div>
+          <div class="card-center">
+              <div class="card-body">
+                  <h1 class="card-title">${card.title}</h1>
+                  <ul class="card-author">
+                      <li class="heading5">CANOPY</li>
+                      <li class="dot"></li>
+                      <li class="author-details">Back End Dev</li>
+                      <li class="dot"></li>
+                      <li class="author-details">2015</li>
+                  </ul>
+                  <p class="card-description">${card.desc}</p>
+              </div>
+              <ul class="card-languages">
+                  <li class="language-item ">${card.cardTechs[0]}</li>
+                  <li class="language-item ">${card.cardTechs[1]}</li>
+                  <li class="language-item ">${card.cardTechs[2]}</li>
+              </ul>
+                  <button id="see-project1" class="see-project btn-card-one" type="button">
+                      See Project
+                  </button>
+          </div>
+      </div>`;
 });
+document.getElementsByClassName('card-section')[0].insertAdjacentHTML('afterbegin', cardSection);
 
-contact.addEventListener('click', () => {
-  menuList.style.display = 'none';
-});
\ No newline at end of file
+// Dynamic Modal
+let modal = '';
+const projectBtn = document.getElementsByClassName('see-project');
+[...projectBtn].forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    modal += `<div id="modal" class="modal">
+    <div id="popup-card" class="popup-card">
+        <div class="popup-heading">
+            <p class="popup-title">${cards[index].title}</p>
+            <i id="cross-popup" class="cross-popup fa-solid fa-x"></i>
```

Comment:
Kindly ensure that when a button is clicked or tapped, the project corresponding to it pops up. Currently, only a particular project pops up.

Below you can see that the project in the background was clicked and a different detail popped up.

![image](https://user-images.githubusercontent.com/20554583/190854198-915b380e-35ba-4d58-b628-d03893b3868b.png)